package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Learn how to use istanbul for tracking test/code coverage in your JS projects",
   "main": "test.js",
   "scripts": {
-    "coverage": "node ./node_modules/.bin/istanbul cover ./test.js",
+    "coverage": "istanbul cover ./test.js",
```

Comment:
@indieforger yeah, Istanbul is a good tool. We prefer to have our `devDependencies` explicit in the `package.json` though and for it to be *clear* which version people are using for the project they are working on. 
That way we never have the situation where someone is using a super old or "*alpha*" version (*globally*) which is incompatible with the project they are working on... We've had that before in the past and took ages to figure out.